lib/graphql/type/abstract_type.ex

```diff
@@ -1,9 +1,12 @@
-defprotocol GraphQL.AbstractType do
+defprotocol GraphQL.Type.AbstractType do
   @type t :: GraphQL.Type.Union.t | GraphQL.Type.Interface.t
 
   @spec possible_type?(GraphQL.AbstractType.t, GraphQL.Type.ObjectType.t) :: boolean
   def possible_type?(abstract_type, object)
 
+  @spec possible_types(GraphQL.AbstractType.t, GraphQL.Schema.t) :: [GraphQL.Type.ObjectType.t]
+  def possible_types(abstract_type, schema)
```

Comment:
Agreed. I'll look into that.  I think the issue is that for one of the compound types (I think `Interface`), you can't tell what the implementations are from the type itself: you need to look at the schema.